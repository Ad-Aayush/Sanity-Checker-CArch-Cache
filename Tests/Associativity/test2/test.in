cache_sim enable ./Tests/Associativity/test2/config.txt
load ./Tests/Associativity/test2/input.s
run
exit

