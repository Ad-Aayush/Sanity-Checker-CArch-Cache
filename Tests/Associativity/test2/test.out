D-cache statistics: Accesses=16, Hit=8, Miss=8, Hit Rate=0.50
